beetsplug/subsonicupdate.py

```diff
@@ -67,10 +76,10 @@ def __create_token():
         return salt, token
 
     @staticmethod
-    def __format_url():
-        """Get the Subsonic URL to trigger a scan. Uses either the url
-        config option or the deprecated host, port, and context_path config
-        options together.
+    def __format_url(endpoint):
+        """Get the Subsonic URL to trigger a endpoint put in paramater.
```

Comment:
Maybe this wording is clearer?
```suggestion
        """Get the Subsonic URL to trigger the given endpoint.
```